pkg/scaleway/sg_root.go

```diff
@@ -68,6 +69,7 @@ func getDetachedSG(instanceAPI *instance.API) ([]ScalewaySecurityGroup, string)
 
 	detachedSgs := []ScalewaySecurityGroup{}
 	for _, SG := range SGs {
+		// do we need to force delete every security group on destroy command ?
 		if SG.UpdateDate.UTC().Add(6*time.Hour).Before(time.Now().UTC()) && !SG.IsDefault && !SG.IsAttached {
```

Comment:
"add time" => be sure that pleco doesn't clean too early (issue is: that kind of object e.g SG, the first time is it created it can be empty)